common/azure-common/src/main/java/com/azure/common/implementation/SwaggerMethodParser.java

```diff
@@ -267,6 +276,37 @@ public String path(Object[] methodArguments) {
         return result;
     }
 
+    /**
+     * Get the encoded form parameters that have been added to this value based on the provided
+     * method arguments.
+     *
+     * @param swaggerMethodArguments the arguments that will be used to create the form parameters'
+     *                               values
```

Comment:
Just as a general style nit: I don't like aligning vertically the 'values' text like this - I would recommend to instead just put values a few spaces indented from the left.